src/index.ts

```diff
@@ -622,7 +623,19 @@ const initialize = (apiKey: string, target: Target, options?: Options): Result =
     eventSourceWithMiddleware = addMiddlewareToEventSource(middleware)
   }
 
-  return { on, off, variation, close, setEvaluations, registerAPIRequestMiddleware }
+  const refreshCache = (): boolean => {
+    // only fetch flags if enough time has elapsed to avoid pressuring backend servers
+    if (lastCacheRefreshTime == null || Math.round((Date.now()-lastCacheRefreshTime)/1000) >= 60) {
+      fetchFlags()
+      lastCacheRefreshTime = Date.now()
```

Comment:
Probably should set this automatically in the `fetchFlags` function instead, otherwise you could kick off a refresh even when the flags were fetched well within the 60s.